rspec

```diff
@@ -0,0 +1,3 @@
+--color
```

Comment:
Proszę usunąć ten plik (poprawna nazwa to `.rspec` a nie `rspec` - ten pierwszy plik już istnieje).